io_scene_vox.py

```diff
@@ -203,21 +224,29 @@ def import_vox(path, *, voxel_spacing=1, voxel_size=1,
         for index in used_palette_indices:
             palette_entry = palette[index]
             material = bpy.data.materials.new("Voxel_mat{}".format(index))
-            material.diffuse_color = [pow(col / 255, gamma_value) for col in palette_entry[:3]]
-            material.diffuse_intensity = 1.0
-            material.alpha = palette_entry[3]
-            material.use_shadeless = use_shadeless
+            material.diffuse_color = [pow(col / 255, gamma_value) for col in palette_entry[:4]]
+            if use_shadeless:
+                material.use_nodes = True
+                material_diffuse_to_emission(material)
             mat_palette.update({index: material})
 
     # peel first voxel information
     voxel, *voxels = voxels
     location = [float(coord) * voxel_spacing for coord in voxel[:3]]
+
     # Using primitive_cube_add once here, to give us a template cube
-    bpy.ops.mesh.primitive_cube_add(radius=0.5 * voxel_size, location=location)
+    bpy.ops.mesh.primitive_cube_add(size=voxel_size, location=location)
```

Comment:
Just create the cube at 0,0,0. It's going to be deleted now anyway
```suggestion
    bpy.ops.mesh.primitive_cube_add(size=voxel_size)
```